src/test/java/step2/model/LottoTest.java

```diff
@@ -0,0 +1,78 @@
+package step2.model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step2.enums.ExceptionMessage;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+/*
+- 당첨번호를 입력받는다.
+- 당첨번호와 비교하여 일치하는 번호의 개수를 출력한다.
+- 당첨번호가 1~45범위에 포함되지 않으면 예외를 발생시킨다.
+- 수익율을 내려준다.
+- 당첨번호를 아직 맞춰보지 않았다면 예외를 발생시킨다.
+*/
+public class LottoTest {
+
+    @DisplayName("당첨번호를 입력받는다.")
+    @Test
+    void addLottoNumTest() {
+        Lotto lotto = new Lotto(1000);
+        lotto.addLotto(Arrays.asList(1, 2, 3, 4, 5, 6));
+        lotto.addLotto(Arrays.asList(7, 8, 9, 10, 11, 12));
```

Comment:
과제는 java 11 기반이니 콜렉션 생성자 사용하시는게 더 편할거에요.

```suggestion
        lotto.addLotto(List.of(1, 2, 3, 4, 5, 6));
        lotto.addLotto(List.of(7, 8, 9, 10, 11, 12));
```